lib/inspec_tools/inspec.rb

```diff
@@ -321,5 +296,87 @@ def populate_groups
       end
       @benchmark.group = group_array
     end
+
+    def generate_title(title, json, date)
+      title ||= "Untitled - Checklist Created from Automated InSpec Results JSON; Profiles: #{json['profiles'].map { |x| x['name'] }.join(' | ')}"
+      title + " Checklist Date: #{date || Date.today.to_s}"
+    end
+
+    def handle_vuln_num(stig_data_list, control)
+      return stig_data_list if control[:vuln_num].nil?
+
+      stig_data_list.push(HappyMapperTools::StigChecklist::StigData.new('Vuln_Num', control[:vuln_num]))
+    end
+
+    def handle_severity(stig_data_list, control)
+      return stig_data_list if control[:impact].nil?
+
+      value = Utils::InspecUtil.get_impact_string(control[:impact])
+      return stig_data_list if value == 'none'
+
+      value = 'high' if value == 'critical'
+
+      stig_data_list.push(HappyMapperTools::StigChecklist::StigData.new('Severity', value))
+    end
+
+    def handle_group_title(stig_data_list, control)
```

Comment:
These methods all just do the same thing effectively:

```ruby
def create_stig_data_element(control, key, value)
    return HappyMapperTools::StigChecklist::StigData.new(value, control[key.to_sym]) unless control[key.to_sym].nil?
end
```

`handle_cci_ref` seems to be the only one that is special.